service/src/main/java/db/postgres/V2_0_1__InitializeIdentityZones.java

```diff
@@ -53,8 +54,8 @@ public void migrate(final JdbcTemplate jdbcTemplate) throws Exception {
     }
 
     private Long createDefaultAuthzZone(final JdbcTemplate jdbcTemplate) {
-        final String insertZoneSql = "INSERT INTO authorization_zone (name, description, subdomain) "
-                + "VALUES (?,?,?)";
+        final String insertZoneSql =
```

Comment:
remember hearing that flyway keeps checksum of these "driver" classes in the db. pls make sure this will not cause any issue during upgrade.